src/app/about/page.tsx

```
@@ -0,0 +1,9 @@
+import About from "@/components/About/About";
+import React from "react";
+
+type pageProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)